adopt-page.html

```diff
@@ -0,0 +1,193 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <title>Adoptadog - Adopt</title>
+  </head>
+
+  <body>
+    <header>
+      <img src="./images/adoptadog-logo.svg" alt="White outline of a dog sitting on its hind legs">
+      <h1>Seattle's premiere dog adoption agency</h1>
+
+      <aside>
+        <img src="./images/cart-icon.svg" alt="White outline of a shopping cart">
+        <p>$1234.56</p>
+        <a href="checkout-page.html">Checkout</a>
+      </aside>
+
+      <nav>
+        <ul>
+          <li><a href="index.html">Home</a></li>
+          <li><a href="adopt-page.html">Dogs</a></li>
+          <li><a href="blog-page.html">Blog</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <main>
+      <section>
+        <h2>Here are all of our Dogs!</h2>
+        <div class="profile-card">
```

Comment:
looking good. You'll want to wrap all these .profile-card divs in a wrapper for layout